src/Umbraco.Web/Mvc/UmbracoAuthorizedController.cs

```diff
@@ -10,5 +16,13 @@
     [UmbracoAuthorize]
     [DisableBrowserCache]
     public abstract class UmbracoAuthorizedController : UmbracoController
-    { }
+    {
+        protected UmbracoAuthorizedController()

```

Comment:
Don't we want to call : `base(....)` and use `Current` to populate the dependencies?

Reply:
It will automagically ca'll the one in UmbracoController. 😋